packages/next-on-pages/src/api/getRequestContext.ts

```diff
@@ -34,7 +34,17 @@ export function getRequestContext<
 		throw new Error('Error: failed to retrieve the Cloudflare request context');
 	}
 
-	return {
-		...cloudflareRequestContext,
-	};
+	return cloudflareRequestContext;
+}
+
+export function getOptionalRequestContext<
+	CfProperties extends Record<string, unknown> = IncomingRequestCfProperties,
+	Context = ExecutionContext,
+>(): RequestContext<CfProperties, Context> | null {
+	try {
+		const cloudflareRequestContext = getRequestContext<CfProperties, Context>();
+		return cloudflareRequestContext;
```

Comment:
```suggestion
		return getRequestContext<CfProperties, Context>();
```

Reply:
I've anyways applied the other suggestion so this doesn't apply anymore 🙂👍